web/themes/custom/pineapple/sass/layouts/views/_view--recipes.scss

```diff
@@ -0,0 +1,101 @@
+.view-recipes {
+
+  form {
+    padding: 0;
+
+    .form--inline {
+      width:100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      align-items: flex-start;
+      margin-bottom: $unit *3;
+
+      @include breakpoint($lap) {
+        display: inline-flex;
+        flex-direction: row;
+        align-items: flex-end;
+      }
+
+      .form-item,
+      .form-actions {
+        margin: 0;
+        padding: 0;
+        margin-bottom: 12px;
+
+        @include breakpoint($lap) {
+          margin-bottom: 0;
+        }
+
+        &.form-item-title {
+          width: 100%;
+
+          @include breakpoint($lap) {
+            width: 20%;
+          }
+        }
+
+        &.form-type-select {
+          width: 100%;
+
+          @include breakpoint($lap) {
+            width: 40%;
+            display: inline-flex;
+          }
+
+          .form-select {
+            padding: 0;
+            margin: 0;
+            margin-top: $unit/2;
+
+            @include breakpoint($lap) {
+              margin-top: 0;
+            }
+
+            &:hover {
+              background-color: $white;
+            }
+
+            option {
+              &:hover {
+                background-color: $gray;
+              }
+            }
+          }
+        }
+
+        &.form-composite {
+          .form-item {
+            display: inline-flex;
+            align-items: center;
+          }
+        }
+
+        input {
+          margin: 0;
+        }
+
+        label {
+          font-size: $small-font;
+          display: flex;
+          align-items: center;
+          margin: 0;
+
+          @include breakpoint($lap) {
+            margin: 0 $unit;
+          }
+        }
+      }
+
+      .button {
+        background-color: $primary;
+        color: white;
```

Comment:
I think the button should be a different colour, because you've set a pattern that card buttons are black, a submit or apply button should be a different colour so that it can have more attention drawn to it

Reply:
Done.